src/github/graphql.ts

```diff
@@ -137,7 +137,7 @@ export interface Commit {
 
 export interface AssignedEvent {
 	__typename: string;
-	databaseId: number;
+	id: number;
```

Comment:
Can you revert this file as whell as queries.gql and utils.ts? They are part of a change that will be in the August milestone.